pkg/healthcheck/healthcheck.go

```diff
@@ -703,6 +704,13 @@ func (hc *HealthChecker) allCategories() []*Category {
 						return
 					},
 				},
+				{
+					description: "Cluster nodes' CIDRs are present within clusterNetworks",
```

Comment:
I think we should move this check to the `linkerd-config` check category given that what we are really doing is checking to see if the `clusterNetworks` field in our config match up with the podCIDR values in each our cluster node.

I would also rewrite the description to reflect this. Something like:
```
`clusterNetworks` config contains all node podCIDRs
```